### ISSUE TITLE
Raw text instead of the mark-up in Wiki

### ISSUE THREAD

**Issue by mipastgt** [palexdev/MaterialFX#139]
The Wiki (https://github.com/palexdev/MaterialFX/wiki) renders the raw text instead of the mark-up for most of the entries.

The `.md` at the end is wrong and I don't know whether spaces are allowed in the file names.
```
* [Buttons](https://github.com/palexdev/MaterialFX/wiki/Buttons)
* [Check Boxes](https://github.com/palexdev/MaterialFX/wiki/Check%20Boxes.md)
* [Combo Boxes](https://github.com/palexdev/MaterialFX/wiki/Combo%20Boxes.md)
* [Date Pickers](https://github.com/palexdev/MaterialFX/wiki/Date%20Pickers.md)
* [Filter Panes](https://github.com/palexdev/MaterialFX/wiki/Filter%20Pane.md)
* [Lists](https://github.com/palexdev/MaterialFX/wiki/Lists.md)
* [Paginations](https://github.com/palexdev/MaterialFX/wiki/Pagination.md)
* [Progress](https://github.com/palexdev/MaterialFX/wiki/Progress.md)
* [Radio Buttons](https://github.com/palexdev/MaterialFX/wiki/Radio%20Button.md)
* [Ripple Generators](https://github.com/palexdev/MaterialFX/wiki/Ripple%20Generator.md)
* [Sliders](https://github.com/palexdev/MaterialFX/wiki/Slider.md)
* [Steppers](https://github.com/palexdev/MaterialFX/wiki/Stepper.md)
* [Tables](https://github.com/palexdev/MaterialFX/wiki/Table%20Views.md)
* [Text Fields](https://github.com/palexdev/MaterialFX/wiki/Text%20Fields.md)
* [Toggles](https://github.com/palexdev/MaterialFX/wiki/Toggles)
* [Tooltips](https://github.com/palexdev/MaterialFX/wiki/Tooltip.md)
```

**Comment by palexdev** [comment]
@mipastgt Ah yes I can confirm the issue. Kinda strange to be honest because when I published the new wiki I tested all the pages and they were working even the ones with space characters. 
I guess recent changes may have broken something. I'm writing the wiki with MarkText because I really like the features but it's "a little bit" buggy (even when saving files).
I'll check the issue and fix it as soon as possible
Thanks for pointing out! 😉
